pkg/x/vmwscript/vm_provision.go

```diff
@@ -392,10 +425,90 @@ func watchPid(ctx context.Context, client *govmomi.Client, vm *object.VirtualMac
 			processTimeout++
 			if processTimeout == 12 { // 12x5 seconds == 60 second time out
 				fmt.Printf("\n")
-				log.Info("Process no longer watched, VMware Tools may have been restarted")
+				log.Warn("Process no longer watched, VMware Tools may have been restarted")
 				break
 			}
 		}
 	}
 	return nil
 }
+
+// setCentosNetwork - CentOS only networking
+func setCentosNetwork(ctx context.Context, client *govmomi.Client, vm *object.VirtualMachine, auth *types.NamePasswordAuthentication, netConfig *NetworkConfig) error {
```

Comment:
Just a comment... nothing for you to do at the moment...

We should look at networking setups for a couple of other platforms and come up with a common interface for this.

Reply:
Yeah that's the plan. I just need to install some more distros and then I can come up with some sane way of presenting a network configuration back to the end user regardless of underlying distro. Not sure if it would be worth doing the same for things like firewall rules as well, but then it depends how far down the hole we want to go. The main reason for this is just to bootstrap a single instance that InfraKit can then make use of to deploy and scale etc.